pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -143,11 +143,11 @@ public ProxyServiceStarter(String[] args) throws Exception {
             // load config file
             config = PulsarConfigurationLoader.create(configFile, ProxyConfiguration.class);
 
-            if (isBlank(metadataStoreUrl)) {
-                // Use zookeeperServers from command line if metadataStoreUrl is empty;
-                config.setMetadataStoreUrl(zookeeperServers);
-            } else {
-                // Use metadataStoreUrl from command line
+            if (!isBlank(zookeeperServers)) {
+                config.setZookeeperServers(zookeeperServers);
```

Comment:
Previously AbstractZkLedgerManager was used to create LedgerManagerFactory. In this issue, MetadataDriver was used to create LedgerManagerFactory, but their uri were different, so the zk test case would fail.